library/src/main/java/com/chuckerteam/chucker/internal/support/ChessboardDrawable.kt

```diff
@@ -0,0 +1,75 @@
+package com.chuckerteam.chucker.internal.support
```

Comment:
I wasn't sure about the package for this class. I added it to the `support` but maybe `ui` is more appropriate.

Reply:
From my point of view it is fine to place in `support`.